pkg/virt-api/webhooks/mutating-webhook/mutators/vmi-mutator.go

```diff
@@ -114,6 +115,32 @@ func (mutator *VMIsMutator) Mutate(ar *v1beta1.AdmissionReview) *v1beta1.Admissi
 	}
 }
 
+func (mutator *VMIsMutator) setDefaultNetworkInterface(obj *v1.VirtualMachineInstance) {
+	autoAttach := obj.Spec.Domain.Devices.AutoattachPodInterface
+	if autoAttach != nil && *autoAttach == false {
```

Comment:
I don't see such a logic removed somewhere else. We have tests which ensure that autoattach happens. Shouldn't there be someting somewhere removed?

Reply:
I'm not sure. I didn't remove the auto-attach logic, I've just moved this method from api/v1/defaults.go to the mutator to rely on the cluster config.